src/sources/internal_logs.rs

```diff
@@ -30,42 +30,47 @@ pub struct InternalLogsConfig {
     /// Set to `""` to suppress this key.
     ///
     /// [global_host_key]: https://vector.dev/docs/reference/configuration/global-options/#log_schema.host_key
-    #[serde(default)]
-    host_key: Option<OptionalValuePath>,
+    #[serde(default = "host_key")]
+    host_key: OptionalValuePath,
 
     /// Overrides the name of the log field used to add the current process ID to each event.
     ///
     /// By default, `"pid"` is used.
     /// Set to `""` to suppress this key.
-    #[serde(default)]
-    pid_key: Option<OptionalValuePath>,
+    #[serde(default = "pid_key")]
+    pid_key: OptionalValuePath,
 
     /// The namespace to use for logs. This overrides the global setting.
     #[configurable(metadata(docs::hidden))]
     #[serde(default)]
     log_namespace: Option<bool>,
 }
 
+fn host_key() -> OptionalValuePath {
+    OptionalValuePath::from(owned_value_path!("host"))
```

Comment:
This isn't technically correct, the default is the value from `log_schema().host_key()`.

I don't remember if this came up before, but it would fall out of sync if we happened to change the global schema.

Reply:
ah, yeah I forgot to change this back. I had it originally coming from the log schema , but was facing an issue in the legacy log schema unit tests. I tried changing it to manually setting here to rule this out but forgot to change back. It ended up being that I needed to implement `Default` and it was kinda a pain to figure out lol. Thanks for catching that